docs/source/index.rst

```diff
@@ -31,12 +31,12 @@ architectures, and common image transformations for computer vision.
    :maxdepth: 2
    :caption: Package Reference
 
+   models
```

Comment:
Somehow the brain of users is accustomed to see docs in the previous order. 
It might be unpleasant to change to order. I love the previous order. And kind off in presentations related to torchvision, E.g. Francisco's at PyTorch day, he presents by the previous order,

Would love the old order :)

Reply:
The old order is alphabetical, and might not reflect the order of what users may be looking for.

I think that users are more interested in models, transforms and ops than say i.o. operations, so I changed the order to reflect the importance of each section. Obviously it's not set in stone and I'm happy to hear other's thoughts, but I don't think we should worry too much about current readers' habits. It's still very obvious which section corresponds to what, so users will get used to this one fast :)

Reply:
So let's arrange according to usage order. Simple LRU caching here :smile: 

Datasets like MNIST, ImageNet are universal, so probably everyone needs it.
Transforms to convert ToTensor etc Are almost in every code.
Models are not used all places.
Ops are then used a bit more scarce.
Utils, are least used I think,

So the order datasets, transforms, models, ops, utils :)

Reply:
Thanks for the input @oke-aditya , I will update accordingly